python/lsst/ts/xml/data/sal_interfaces/TunableLaser/TunableLaser_Commands.xml

```diff
@@ -43,4 +43,8 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_command_trigger</EFDB_Topic>
```

Comment:
Please consider `triggerBurst` for clarity.

Please add a Description.

Reply:
Also please consider combining the setBurstMode and setBurstCount commands into a single setBurstMode command with a `burstCount` field. That would give the user more predictable behavior -- when they enable burst mode they know exactly how many pulses they will get per trigger.